app/models/robots.py

```diff
@@ -0,0 +1,49 @@
+from app.models import *
+
+
+class UserRobot(Base, UserMixin):
+    __tablename__ = "user_robot"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    name = Column(String(20), nullable=False)
+    image_file = Column(String(40), nullable=False, default="default.jpg")
+    location = Column(Geography(geometry_type='POINT', srid=4326))
+    domicile_geolocation = Column(Geography(geometry_type='POINT', srid=4326))
+    messages = relationship("RobotMessages", backref="author", lazy=True)
+    profile_robot = relationship(
+        "RobotProfile", uselist=False, back_populates="user_robot"
+    )
+
+    def __repr__(self):
+        return f"UserRobot('{self.name}', '{self.image_file}')"
+
+
+class RobotMessages(Base, UserMixin):
+    __tablename__ = "messages_robot"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    name = Column(String(20), nullable=False)
+    message = Column(String(400), nullable=False)
+    user_id = Column(Integer, ForeignKey("user_robot.id"), nullable=False)
+
+    def __repr__(self):
+        return f"RobotMessages('{self.message}')"
+
+
+class RobotProfile(Base, UserMixin):

```

Comment:
I don't see much sense in having a separate profile model for both User an Robot